src/helm/benchmark/run_specs.py

```diff
@@ -403,6 +404,33 @@ def get_machine_translation_adapter_spec(
     )
 
 
+def get_vlm_generation_adapter_spec(
```

Comment:
We should really break up this file into multiple ones - can we start doing that with this PR?